internal/compiler/object_tree.rs

```diff
@@ -251,9 +251,12 @@ pub struct Component {
     /// we can preserve the order across multiple inlining passes.
     pub inlined_init_code: RefCell<BTreeMap<usize, Expression>>,
 
-    /// Code to be inserted into the constructor, such as font registration, focus setting or
-    /// init callback code collected from elements.
-    pub init_code: RefCell<Vec<Expression>>,
+    // Code from init callbacks collected from elements
+    pub constructor_code: RefCell<Vec<Expression>>,
+    /// Code to set the initial focus via forward-focus on the Window
+    pub focus_setting_code: RefCell<Vec<Expression>>,
+    /// Code to register embedded fonts.
+    pub font_registration_code: RefCell<Vec<Expression>>,
```

Comment:
I'd admit I'm a bit concerned seeing this kind of details in this part of the code. 
But I guess that's just how it is.

Would it make sense to have a `struct InitCode { ... }`  maybe?

Reply:
Good idea! I'll do that.